import_test_data/run_imports.sh

```diff
@@ -0,0 +1,362 @@
+#! /bin/sh
+
+if [ -x "bin/run" ]; then
+  FAUNA_CMD="bin/run"
+elif [ -x "../bin/run" ]; then
+  FAUNA_CMD="../bin/run"
+else
+  echo "Can't find the fauna command (tried bin/run and ../bin/run)"
+  exit 1
+fi
+
+# Uncomment to default to the current impl
+#FAUNA_CMD="fauna"
+
+echo "Using following to execute: $FAUNA_CMD"
+
+cleanup_collection () {
+  COLLECTION="$1"
+  if [ -z "$COLLECTION" ]; then
+    echo "missing collection arg"
+    exit 1
+  fi
+  
+
+  $FAUNA_CMD eval --stdin --endpoint data-import-test &> /dev/null << CMD
+    If(
+      Exists(Collection("$COLLECTION")),
+      Map(
+        Paginate(Documents(Collection("$COLLECTION"))),
+        Lambda("X",Delete(Var("X")))
+      ),
+      "$COLLECTION does not exist. Import job will create it"
+    )
+CMD
+}
+
+ensure_db () {
+  DB="$1"
+  if [ -z "$DB" ]; then
+    echo "missing db arg"
+    exit 1
+  fi
+  
+
+  $FAUNA_CMD eval --stdin --endpoint data-import-test &> /dev/null << CMD
+    If(
+      Exists(Database("db")),
+      "Database exists!",
+      CreateDatabase({name: "$DB"})
+    )
+CMD
+}
+
+cleanup_all_collections () {
+  cleanup_collection "bool_type"
+  cleanup_collection "date_type"
+  cleanup_collection "bad_date_type"
+  cleanup_collection "number_type"
+  cleanup_collection "bad_number_type"
+  cleanup_collection "auto_type_translation"
+  cleanup_collection "default_null_inference"
+  cleanup_collection "short_rows"
+  cleanup_collection "short_rows_empty_strings"
+  cleanup_collection "short_rows_with_type_translations"
+  cleanup_collection "too_long_rows"
+  cleanup_collection "headers"
+  cleanup_collection "alt_char_headers"
+  cleanup_collection "json_array"
+  cleanup_collection "json_nested_type_trans"
+  cleanup_collection "multi_array"
+  cleanup_collection "mixed_array_and_l"
+  cleanup_collection "json_l"
+  cleanup_collection "mixed_l_and_array"
+  cleanup_collection "foo"
+}
+
+fail_test() {
+    MESSAGE=$1
+    echo "\033[31m TEST FAILED: $MESSAGE \033[0m" >&2
+    exit 1
+}
+
+run_type_tests () {
+  cleanup_collection "bool_type"
+  $FAUNA_CMD import --endpoint data-import-test --type=favorite::bool --path=type_tests/bool_type.csv
+  if [ $? != 0 ];then
+    fail_test "bool_type.csv didn't import with success"
+  fi
```

Comment:
I thought I heard there was some effort to read out data from these tests. I don't see that in this change. Did I miss it?

Reply:
i added a bunch of tests to in the node package itself.

those tests read data and confirm equality

Reply:
i left these as truly scrappy.